go/adbc/driver/flightsql/flightsql_connection.go

```diff
@@ -64,10 +64,16 @@ func doGet(ctx context.Context, cl *flightsql.Client, endpoint *flight.FlightEnd
 	}
 
 	var (
-		cc interface{}
+		cc          interface{}
+		hasFallback bool
 	)
 
 	for _, loc := range endpoint.Location {
+		if loc.Uri == flight.LocationReuseConnection {
+			hasFallback = true
+			continue
+		}
```

Comment:
shouldn't we `break` instead of `continue`? or at least do the same as we do below and attempt to reuse the connection, continuing the loop if it errors, returning if it succeeds?

Reply:
is that correct semantically? We don't want to maintain the order and have it do the reuse-connection before moving on to further locations?

Reply:
I think it's always been said that order is unspecified.

Reply:
Okay then, I'm fine with this :)